dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/AssignmentNode.java

```diff
@@ -84,10 +85,12 @@ public AssignmentNode(Tree tree, Node target, Node expression, boolean synthetic
      *
      * @return the left-hand-side of the assignment
      */
+    @Pure
     public Node getTarget() {
         return lhs;
     }
 
+    @Pure
```

Comment:
The CI failure is because of the missing javadoc for this method. Simply add something similar to what is on `getTarget`.

Reply:
Sure